.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -218,9 +218,10 @@ export async function pickTestGroupRunOrder() {
       : ['build'];
 
   const { defaultQueue, ftrConfigsByQueue } = getEnabledFtrConfigs(FTR_CONFIG_PATTERNS);
-  if (!LIMIT_CONFIG_TYPE.includes('functional')) {
-    ftrConfigsByQueue.clear();
-  }
+
+  const ftrConfigsIncluded = LIMIT_CONFIG_TYPE.includes('functional');
+
+  if (!ftrConfigsIncluded) ftrConfigsByQueue.clear();
```

Comment:
I would love to see a short comment explaining the logic.

Reply:
Ohhh, I think I get the ask now....

Reply:
Why do we clear here

Reply:
Added a comment for the first question.

Reply:
Ok, for the second question, I left what was there.

But to answer more fully, you can see a case where the map [**could** have a size greater than 0](https://github.com/elastic/kibana/blob/cc-drop-ftr-run-order/.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts#L148).